bindings/dd/register_vector_dds.cpp

```
@@ -103,7 +128,19 @@ void registerVectorDDs(const py::module& mod) {
                        true, formatAsPolar);
       },
       "filename"_a, "colored"_a = true, "edge_labels"_a = false,
-      "classic"_a = false, "memory"_a = false, "format_as_polar"_a = true);
+      "classic"_a = false, "memory"_a = false, "format_as_polar"_a = true,
```

Cppcheck (2.18.0)

[error] Boolean value assigned to pointer.

(assignBoolToPointer)